### PR TITLE
Allow pre-created secrets for external database credentials

### PR DESCRIPTION
**Pull request by robbie-demuth**
#### What this PR does / why we need it:

Previously, the chart required that users set the password for an external database via a value. This change allows users to instead provide the name of a pre-created secret containing the password

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.11.0
+version: 0.11.1
```
Comment (rjkernick) [inline stable/sonarqube/Chart.yaml]: Please bump the minor version as this adds a feature

File: stable/sonarqube/values.yaml [hunk 2]
```diff
@@ -148,6 +151,9 @@ mysql:
   # To use an external mySQL instance, set enabled to false and uncomment
   # the line below:
   # mysqlServer: ""
+  # To use an external secret for the password for an external mySQL instance,
```
Comment (rjkernick) [inline stable/sonarqube/values.yaml:160]: Please specify that enabled must be set to false for this to work

File: stable/sonarqube/values.yaml [hunk 3]
```diff
@@ -132,6 +132,9 @@ postgresql:
   # To use an external PostgreSQL instance, set enabled to false and uncomment
   # the line below:
   # postgresServer: ""
+  # To use an external secret for the password for an external PostgreSQL
```
Comment (rjkernick) [inline stable/sonarqube/values.yaml:140]: Please specify that enabled needs to be set to false

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by rjkernick** [comment]
Please merge the latest master branch and increment the version in Chart.yaml

**Comment by robbie-demuth** [comment]
Done, @rjkernick

**Comment by rjkernick** [comment]
One got merged ahead of you. Please merge the latest master and increment the version again.

**Comment by robbie-demuth** [comment]
Done, @rjkernick

**Pull request merged**